docs/sources/datasources/elasticsearch/query-editor/index.md

```diff
@@ -15,64 +15,144 @@ labels:
+  - filter - see [Filter aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-filter-aggregation.html).
+  - geo hash grid - see [Geohash grid aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-geohashgrid-aggregation.html).
+  - date histogram - for time series queries. See [Date histogram aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-datehistogram-aggregation.html).
+  - histogram - Depicts frequency distributions. See [Histogram aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-histogram-aggregation.html).
+  - nested (experimental) - See [Nested aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-nested-aggregation.html).
 
-## Name a time series
+- **@timestamp** - Depicts timestamp data options. Set this when configuring the Elasticsearch data source in the **Time field name** under the [**Elasticsearch details**](/docs/grafana/latest/datasources/elasticsearch/configure-elasticsearch-data-source/#elasticsearch-details) section.
 
-You can control the name for time series via the `Alias` input field.
+- **Interval** - Group by a type of interval. There are option to choose from the dropdown menu to select seconds, minutes, hours or day. You can also add a custom interval such as `30d` (30 days). `Auto` is the default option.
 
-| Pattern              | Replacement value                      |
-| -------------------- | -------------------------------------- |
-| `{{term fieldname}}` | Value of a term group-by               |
-| `{{metric}}`         | Metric name, such as Average, Min, Max |
-| `{{field}}`          | Metric field name                      |
+  - **Min doc count** - The minimum amount of data to include in your query. The default is `0`.
+  - **Thin edges** - Select to trim edges on the time series data points. The default is `0`.
+  - **Offset** - Changes the start value of each bucket by the specified positive(+) or negative (-) offset duration. Examples include `1h` for 1 hour, `5s` for 5 seconds or `1d` for 1 day.
+  - **Timezone** - Select a timezone from the dropdown menu. The default is `Coordinated universal time`.
 
-## Control pipeline metrics visibility
+Click the **+ sign** to add multiple group by options. The data will grouped in order (first by, then by).
 
-Some metric aggregations, such as _Moving Average_ and _Derivative_, are called **Pipeline** aggregations.
-Elasticsearch pipeline metrics must be based on another metric.
-Use the eye icon next to the metric to prevent metrics from appearing in the graph.
-This is useful for metrics you only have in the query for use in a pipeline metric.
+{{< figure src="/static/img/docs/elasticsearch/group-by-then-by-10.2.png" max-width="850px" class="docs-image--no-shadow" caption="Group by options" >}}
```

Comment:
All of these options (starting with time field and ending with timezone) are specifically for `Date Histogram` , so if we want to add documentation for them, I think they should be under `date histogram`. 

Different types of bucket aggregations have different options. So for example `terms` accepts any field (not timestamp) and has different options such as `Order`, `Size`, ...
<img width="1040" alt="image" src="https://github.com/grafana/grafana/assets/30407135/8350f67c-74b3-4f7f-a760-e649feafd3d3">

Reply:
I will add options for each type of group by (bucket aggregation) option - I totally missed this so thank you for pointing this out. :-)

Reply:
added group by/bucket aggregations fileds